helmfiles/lagoon-monitoring/values.yaml

```diff
@@ -0,0 +1,26 @@
+namespace: lagoon-monitoring
+auth:
+  enabled: false
+  host: auth.monitoring.example.com
+  config:
+    vouch:
+      domains: []
+      allowAllUsers: false
+      whiteList: []
+      jwt:
+        secret: super-secret-stuff
```

Comment:
So is the idea that this file is just example values that any user of the `helmfile.yaml` would need to override? If so maybe we could add a comment at the top?

Reply:
ah yes, will add a comment :)